app/my-bets/page.tsx

```diff
@@ -7,96 +7,151 @@ import { Market, Position, tradesOutcomeBalance } from '@/entities';
 
 import { getUserPositions } from '@/queries/conditional-tokens';
 import { UserPosition } from '@/queries/conditional-tokens/types';
-import { getConditionMarket, getMarketUserTrades } from '@/queries/omen';
-import { useQuery, useQueryClient } from '@tanstack/react-query';
-
-import { PropsWithChildren, ReactNode, useCallback, useEffect, useState } from 'react';
+import {
+  Condition,
+  FixedProductMarketMaker,
+  FpmmTrade,
+  getConditionMarket,
+  getMarketUserTrades,
+} from '@/queries/omen';
+import { useQuery, useQueries } from '@tanstack/react-query';
+
+import { PropsWithChildren, ReactNode, useEffect, useMemo, useState } from 'react';
 import { TabBody, TabGroup, TabHeader, TabPanel, TabStyled } from '@swapr/ui';
 import { useAccount } from 'wagmi';
 
+interface UserPositionAll extends UserPosition {
```

Comment:
I think we can improve the name. Adding `All` doesn't give much about the type. What about `UserCompletePositionInfo`, `UserPositionCondition`, `ConditionUserPosition`?

Reply:
the idea was having user position and "all" the extra related data, but I agree we can improve.

This is a `UserPosition` with `fpmmTrades`, `condition` and `market` data.

Maybe complete is a good description.

i think `UserPositionComplete` can work